visual-e2e/examples/test/specs/simple.spec.js

```diff
@@ -0,0 +1,15 @@
+describe('Visual rendering', () => {
```

Comment:
I'm going to update all examples to use the `async` API because as of NodeJS 16, which is the _default_ now, `sync` is not supported anymore.
I will update all examples to `async` and create a separate example for sync that will can be used as an example for all other samples. Can you thus update this to use `async`?